.terraform/modules/34fda75a37d4cc8fcebdb106c425c61d/baikonur-oss-terraform-aws-iam-nofile-11a0b7b/.gitignore

```diff
@@ -0,0 +1,20 @@
+### macOS
```

Comment:
Hmm, it seems that files under .terraform are included in this PR. Please remove them by running `git rm --cached`?

Reply:
done